examples/solids/qfunctions/FSCurrent-NH2.h

```diff
@@ -44,7 +44,7 @@ struct Physics_private {
 // -----------------------------------------------------------------------------
 #ifndef LOG1P_SERIES_SHIFTED
 #define LOG1P_SERIES_SHIFTED
-static inline CeedScalar log1p_series_shifted(CeedScalar x) {
+static CEED_QFUNCTION_HELPER CeedScalar log1p_series_shifted(CeedScalar x) {
```

Comment:
a third ordering

I think we can put the `static` inside the definition of `CEED_QFUNCTION_HELPER` and always write `CEED_QFUNCTION_HELPER return_type function_name(...)`.